handler/handler.go

```diff
@@ -363,7 +363,7 @@ func (h *InstanceComplete) GetS3ContentLength(ctx context.Context, e *event.Expo
 // UpdateInstance updates the instance downlad CSV link using dataset API PUT /instances/{id} endpoint
 // if the instance is published, then the s3Url will be set as public link and the instance state will be set to published
 // otherwise, a private url will be generated and the state will not be changed
-func (h *InstanceComplete) UpdateInstance(ctx context.Context, e *event.ExportStart, size int, isPublished bool, s3Url string) error {
+func (h *InstanceComplete) UpdateInstance(ctx context.Context, e *event.ExportStart, size int, isPublished bool, s3Url string, filename string) error {
```

Comment:
You don't need two `string` declarations here 
```suggestion
func (h *InstanceComplete) UpdateInstance(ctx context.Context, e *event.ExportStart, size int, isPublished bool, s3Url, filename string) error {
```